foundry/pixi/containers/canvasLayers/placeablesLayers/notesLayer.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * The Notes Layer Container
+ */
+declare class NotesLayer extends PlaceablesLayer<Note> {
+  /** @override */
+  static get layerOptions(): PlaceablesLayer.LayerOptions;
```

Comment:
Overridden defaults should be documented with a class specific options type.

Reply:
Does that really add all that much value?

Reply:
Personally I don't think it's strictly necessary. Doesn't hurt but also doesn't gain much. From my perspective it's not needed. But I'm not the repo owner, so I guess in the what @kmoschcau wants counts.

Reply:
If you guys think it doesn't add much value, then this is another point where we should decide on yes or no. I have no problem with being overruled here. But it should be consistent. We added those in other places in the past.

Reply:
I think in situations where the types are changed (e.g. options that might be `null` or `undefined` get a proper default value), we should definitely do this. If it's only to provide the default values, I don't think it's worth a lot. The only benefit is that when somebody is overriding the `getDefaultOptions` method and calling the super `getDefaultOptions`, he could see the default values in the TS doc, so he knows what values he is overriding. That is _some_ value. Maybe we should just do it... I wouldn't go over old classes we have already covered that don't use this pattern (of which there are quite a few because I for once didn't use this pattern at all until now).

Reply:
What about documenting default values like this:
```ts
 /**
  * @override
  * @defaultValue
  * ```
  * {
  *   canDelete: game.user.isGM,
  *   canDragCreate: false,
  *   controllableObjects: false,
  *   gridPrecision: 2,
  *   objectClass: Note,
  *   quadtree: false,
  *   rotatableObjects: false,
  *   sheetClass: NoteConfig,
  *   snapToGrid: true,
  *   sortActiveTop: true,
  *   zIndex: 60
  * }
  * ```
  */
  static get layerOptions(): PlaceablesLayer.LayerOptions;
```

Reply:
Also fine with me.

Reply:
In discord we came to the conclusion that we will write it as a single `@defaultValue` tag into the method documentation. Only the explicitly overwritten parameters will be written out in that code block. Additionally, the code statements that merge the overwritten options with the ones from the superclass will also be added. For most methods this essentially means to just copy paste the method content into the code block.